lib/rules/no-unnecessary-act.ts

```diff
@@ -0,0 +1,141 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { createTestingLibraryRule } from '../create-testing-library-rule';
+import {
+  getDeepestIdentifierNode,
+  getStatementCallExpression,
+  isEmptyFunction,
+} from '../node-utils';
+
+export const RULE_NAME = 'no-unnecessary-act';
+export type MessageIds =
+  | 'noUnnecessaryActTestingLibraryUtil'
+  | 'noUnnecessaryActEmptyFunction';
+
+export default createTestingLibraryRule<[], MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'Disallow wrapping Testing Library utils or empty callbacks in `act`',
+      category: 'Possible Errors',
+      recommendedConfig: {
+        dom: false,
+        angular: false,
+        react: false, // this should be enabled on v5 of the plugin
```

Comment:
maybe we could create a ticket/discussion for V5, so this kind of stuff doesn't get lost in comments. Not because there is any release date in mind, but as a planning  (and documenting) session

If you agree and do that, then this comment can be removed

Reply:
Yes! I have some notes written about things to do in a future v5. I'll try to create a proper project soon, so I'm gonna remove this comment.